docker/Dockerfile_db

```diff
@@ -1,4 +1,4 @@
-FROM candidate-registry.fedoraproject.org/postgresql
+FROM registry.fedoraproject.org/f29/postgresql
```

Comment:
I wonder why the image isn’t updated anymore. FWIW, it’s also in the candidate registry: https://candidate-registry.fedoraproject.org/v2/_catalog, but they’re identical, so shouldn’t matter if we’re targeting a specific release.